src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx

```diff
@@ -0,0 +1,276 @@
+import React from "react";
+import { Card, Modal, Row, Col, Button, Form } from "antd";
+import Icon, {
+  CloseCircleOutlined,
+  PlusCircleOutlined,
+} from "@ant-design/icons";
+import styles from "./DelegatedSupportedCamps.module.scss";
+export default function DelegatedSupportCampsUI({
+  RemoveCardDelegatedSupportedCamps,
+  handleSupportedCampsCancel,
+  isRemoveSupportModalVisible,
+  handelViewMoreModalCancel,
+  showViewMoreModal,
+  viewMoreModalVisible,
+}) {
+  function CardTitle(props) {
+    return (
+      <div className={styles.card_heading_title}>
+        For topics<span> "{props.value}"</span>
+      </div>
+    );
+  }
+  return (
+    <div>
+      <Card
+        className={styles.cardBox_tags}
+        type="inner"
+        size="default"
+        title={<CardTitle value="Theories of Consciousness" />}
+        extra={
+          <a
+            className={styles.RemoveCardSupported}
+            onClick={() => RemoveCardDelegatedSupportedCamps()}
+          >
+            <CloseCircleOutlined /> Remove support{" "}
+          </a>
+        }
+        style={{ width: 760, marginBottom: 16 }}
+      >
+        <div>
+          <Row>
+            <Col span={12}>
+              <div className={styles.line_height}>
+                <p>
+                  Support delegated to:
+                  <span className={styles.Bluecolor}>Pranav</span>
+                </p>
+                <p>
+                  Nick Name:
+                  <span className={styles.Bluecolor}>rohit_telentelgia</span>
+                </p>
+              </div>
+            </Col>
+            <div></div>
+            <Col span={12} className={styles.border_left}>
+              <div className={styles.line_height1}>
+                <p>
+                  <b>Current Supported Camps:</b>
+                </p>
+                <p>
+                  1. &nbsp;
+                  <span className={styles.Bluecolor}>
+                    Technological Improvement
+                  </span>
+                </p>
+                <p>
+                  2. &nbsp;
+                  <span className={styles.Bluecolor}>Digital Identity</span>
+                </p>
+                <p>
+                  3. &nbsp;<span className={styles.Bluecolor}>Prototype</span>
+                </p>
+              </div>
+              <a className={styles.mrgn_left} onClick={showViewMoreModal}>
+                View More
+              </a>
+            </Col>
+          </Row>
+        </div>
+      </Card>
+      <Card
+        className={styles.cardBox_tags}
+        type="inner"
+        size="default"
+        title={<CardTitle value="Front End Language" />}
+        extra={
+          <a
```

Comment:
Not sure if we can use Link here. Can you please check

Reply:
Done